### PR TITLE
[FEATURE] Add option to exit on issues

### PR DESCRIPTION
**Pull request by GabrielCastro**
##### Description

Add a new cli flag `-e` to produce a non-zero exit code if there are any issues
This makes bash scripts easier t write

##### Type of change

- [x] New feature :sparkles:

##### How Has This Been Tested?

I've tested this manually, but I'm not sure what the best way to automate this would be

##### Checklist:

- [x] I have :tophat: my changes (A 🎩 specifically includes pulling down changes, setting them up, and manually testing the changed features and potential side effects to make sure nothing is broken)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The test coverage did not decrease

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -55,7 +56,8 @@ func init() {
 	RootCmd.PersistentFlags().BoolVarP(&rootConfig.allPods, "allPods", "a", false, "Audit againsts pods in all the phases (default Running Phase)")
 	RootCmd.PersistentFlags().StringVarP(&rootConfig.namespace, "namespace", "n", apiv1.NamespaceAll, "Specify the namespace scope to audit")
 	RootCmd.PersistentFlags().StringVarP(&rootConfig.manifest, "manifest", "f", "", "yaml configuration to audit")
-	RootCmd.PersistentFlags().StringVarP(&rootConfig.auditConfig, "auditconfig", "k", "", "filepath for kubeaudit config file")
+	ootCmd.PersistentFlags().StringVarP(&rootConfig.auditConfig, "auditconfig", "k", "", "filepath for kubeaudit config file")
```
Comment (nschhina) [inline cmd/root.go]: typo resulting in missing leading R

File: cmd/util.go [hunk 2]
```diff
@@ -336,9 +336,14 @@ func runAudit(auditFunc interface{}) func(cmd *cobra.Command, args []string) {
 			log.Error(err)
 			return
 		}
+		occurrenceCount := 0
 		results := getResults(resources, auditFunc)
 		for _, result := range results {
 			result.Print()
+			occurrenceCount += len(result.Occurrences)
```
Comment (nschhina) [inline cmd/util.go]: I don't think we want to return 1 in the case where kubeaudit returns a bunch of Occurrences with level `warn` or `info`, maybe you should strictly check for level `error` in this case, using a bool flag in place of occurrenceCount is much better.

**Comment by nschhina** [comment]
Hi @GabrielCastro, just checking on how you're doing with this PR? Let me know if you need any help!

**Comment by genevieveluyt** [comment]
I'm closing this due to inactivity. @GabrielCastro feel free to re-open if you want to keep working on it.

**Pull request closed without merging**